libraries/rush-lib/src/cli/actions/InstallAction.ts

```diff
@@ -45,6 +46,11 @@ export class InstallAction extends BaseInstallAction {
       parameterLongName: '--check-only',
       description: `Only check the validity of the shrinkwrap file without performing an install.`
     });
+
+    this._resolutionOnlyParameter = this.defineFlagParameter({
+      parameterLongName: '--resolution-only',
+      description: `Only perform dependency resolution, useful for ensuring peer dependendencies are up to date.`
```

Comment:
```suggestion
      description: `Only perform dependency resolution, useful for ensuring peer dependendencies are up to date. Note that this flag is only supported when using the pnpm package manager.`
```

You can also consider only initializing this flag if pnpm is used.

Reply:
updated to only initialize the flag if pnpm is used in [0226e5f](https://github.com/microsoft/rushstack/pull/4893/commits/0226e5fb1e87e6c3f638b68eaa470f4f8dd7a8d1)